execution/execution.go

```diff
@@ -75,6 +76,6 @@ func (execution *Execution) Fail(err error) error {
 // GetDuration returns the duration of the execution in seconds
 func (execution *Execution) GetDuration() int64 {
 	delta := execution.Stop - execution.Start
-	deltaInSecond := float64(delta) / 1e9
+	deltaInSecond := float64(delta) / float64(time.Second.Nanoseconds())
```

Comment:
For information on the proto I tried:

###  cast type:
`(gogoproto.casttype) = "time"` doesn't work because proto/gogo doesn't integrate default packages.

### google proto 
https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp 
This proto doesn't implement the equal function that we need 

Also, the time is hard to set from convert from int64, the only way is to extract the number of seconds and the left of nanoseconds...